src/getUTCOffsetFromDate.ts

```diff
@@ -0,0 +1,10 @@
+import dayjs from 'dayjs';
+import {isString} from 'lodash';
+
+export default (date: string | Date): string => {
+  if (isString(date)) {
```

Comment:
Better to always pass to dayjs. 
Eg. if not full date strinbg will be provided, like `2019-01-25T00:00:00`